DRAFT_RELEASE_NOTES.md

```diff
@@ -33,6 +33,7 @@ To reduce burden of upgrading aiSSEMBLE, the Baton project is used to automate t
 | upgrade-tiltfile-aissemble-version-migration       | Updates the aiSSEMBLE version within your project's Tiltfile                                                                                                                 |
 | upgrade-v2-chart-files-aissemble-version-migration | Updates the Helm chart dependencies within your project's deployment resources (`<YOUR_PROJECT>-deploy/src/main/resources/apps/`) to use the latest version of the aiSSEMBLE |
 | upgrade-v1-chart-files-aissemble-version-migration | Updates the docker image tags within your project's deployment resources (`<YOUR_PROJECT>-deploy/src/main/resources/apps/`) to use the latest version of the aiSSEMBLE       |
+| hive-universal-config-yaml-migration               | Updates hive-metastore-service values.yaml to removes hive username to use values from configuration store service only if hive username has default value.                  |
```

Comment:
```suggestion
| hive-universal-config-yaml-migration               | Removes the default hive username (if present) from hive-metastore-service values.yaml so that it can be set by the configuration store service         |
```